<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabExtensions</class>
 <widget class="QWidget" name="tabExtensions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>150</number>
     </property>
     <item>
      <widget class="QLabel" name="label_extensions_settings">
       <property name="text">
        <string>Extensions settings:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_what_is_extensions">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Extensions allows you to customize settings for a certain filetype. For example all .webm files can automatically loop, while any other filetype does not.&lt;/p&gt;&lt;p&gt;Since there's a plethora of settings you can add here, it has been sat as a text editor instead of a bunch of checkboxes. Check the settings around in the configurator for settings tips. You can also check the manual for further settings.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>(?)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="webm">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>EXAMPLE</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_name_webm">
           <property name="text">
            <string>.exmpl</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPlainTextEdit" name="webm_text">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string>ontop
loop-file=yes</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_note_will_not_be_added">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Note: This example extension will not be added to the config.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>30</number>
     </property>
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLineEdit" name="valuebox_add_new_extension"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="button_add_extension">
       <property name="text">
        <string>Add Extension</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="button_remove_current">
       <property name="text">
        <string>Remove Current</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>horizontalSpacer</zorder>
  <zorder>horizontalSpacer_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
